cli/server.go

```diff
@@ -137,6 +139,7 @@ func server() *cobra.Command {
 	cmd.Flags().StringVar(&artifactory, "artifactory", "", "Artifactory server URL.")
 	cmd.Flags().StringVar(&repo, "repo", "", "Artifactory repository.")
 	cmd.Flags().StringVar(&address, "address", "127.0.0.1:3001", "The address on which to serve the marketplace API.")
+	cmd.Flags().IntVar(&listcacheduration, "list-cache-duration", 1, "The duration of the extension cache in minutes.")
```

Comment:
Great idea to add a flag.  What do you think about making `listcacheduration` a `time.Duration` directly (and use `DurationVar`) so users can type things like `1h` or `20m`?

Reply:
That's a great idea. :slightly_smiling_face:  I simply did not think of the possibility, but it's an improvement both code wise and usability wise, so I'll make that change.